tests/lib/ansible_helper.py

```diff
@@ -153,14 +153,15 @@ def create_inventory(self, hardware):
 
     def run_play_raw(self, playbook):
         path = os.path.abspath(os.path.join(
-            os.path.dirname(__file__), 'assets/ansible', playbook
+            os.path.dirname(__file__), '../assets/ansible', playbook
         ))
         logger.info(f'Running playbook {path}')
         try:
             subprocess.run(['ansible-playbook', '-i', self.inventory_dir,
-                            path])
+                            path], check=True)
         except subprocess.CalledProcessError:
-            logger.error('An error occured executing Ansible playbook')
+            logger.exception('An error occured executing Ansible playbook')
+            Exception("An error occurred running playbook")
```

Comment:
Can we raise the original exception here instead of creating a new one with less information?

Reply:
Changed statements to logger.exception (which does add trace) and raise statement